blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineUpdateRequest.java

```diff
@@ -55,6 +55,7 @@ public BluePipeline update(BluePipeline pipeline) throws IOException {
                 folder.scheduleBuild(new Cause.UserIdCause());
             }
         }
+        item.save();
```

Comment:
Maybe we should but better to check with @kzantow, last time I discussed with @kzantow on this, he thought there might be some use cases where we want to update existing orgs outside of create flow.